frontend/src/concepts/roleBinding/RoleBindingPermissionsTableRowAdd.tsx

```diff
@@ -42,7 +48,8 @@ const ProjectSharingTableRowAdd: React.FC<ProjectSharingTableRowPropsAdd> = ({
           />
         </Td>
         <Td dataLabel="Permission">
-          <ProjectSharingPermissionSelection
+          <RoleBindingPermissionsPermissionSelection
+            permissionOptions={permissionOptions}
```

Comment:
You probably don't want to show this selection dropdown if there's only one option, just like when editing the regular `RoleBindingPermissionsTableRow`.

Reply:
kept forgetting that edit/add were two separate things :p